packages/catalog-model/src/entity/ref.ts

```diff
@@ -86,25 +86,25 @@ export function parseEntityName(
  */
 export function parseEntityRef(
   ref: EntityRef,
-  context?: { defaultKind: string },
```

Comment:
these are just a reordering of the signatures, so the most precise is at the top; otherwise TS sometimes picks the wrong one